evaluator/evaluator.py

```diff
@@ -272,39 +302,30 @@ def process_message(message):
                     LOGGER.error("Received message is missing type field: %s", msg_dict)
                     return
                 if msg_dict['type'] in ['upload_new_file', 're-evaluate_system']:
-                    with DatabasePoolConnection() as conn:
-                        with conn.cursor() as cur:
-                            LOGGER.info("Received message type: %s", msg_dict['type'])
-                            # Lock the system for processing
-                            cur.execute("""SELECT inventory_id, vmaas_json, rh_account,
-                                                opt_out FROM system_platform
-                                        WHERE inventory_id = %s
-                                        FOR UPDATE""", (msg_dict['system_id'],))
-                            system_platform = cur.fetchone()
-                            if system_platform is not None:
-                                try:
-                                    self.evaluate_vmaas(system_platform, cur)
-                                    conn.commit()
-                                except DatabaseError:
-                                    LOGGER.exception("Unable to store data: ")
-                                    conn.rollback()
-                            else:
-                                INV_ID_NOT_FOUND.inc()
-                                LOGGER.error("System with inventory_id not found in DB: %s", msg_dict['system_id'])
+                    process_func = self.process_upload_or_re_evaluate
+                elif 'system_id' not in msg_dict or msg_dict["system_id"] is None:
```

Comment:
this check is on wrong place and is not needed actually, only our components (vmaas-sync and listener produce messages for evaluator) and system_id is always defined

please remove it